src/components/BoardReactionMenu/BoardReactionMenu.scss

```diff
@@ -0,0 +1,100 @@
+@import "src/constants/style";
+
+.board-reactions__root {
+  position: absolute;
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  bottom: $column__border-width;
+}
+
+.board-reactions__container {
+  width: 374px;
+  height: 64px;
+
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  gap: 8px;
+
+  background-color: $color-white-one;
+  border-radius: $rounded--large;
+  box-shadow: $box-shadow--light;
+
+  padding: 0 $padding--default;
+
+  user-select: none;
+}
+
+.board-reactions__item {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  cursor: pointer;
+}
+
+.board-reactions__reaction {
+  all: unset; // reset button
+  cursor: pointer;
+
+  width: 54px;
+  height: 54px;
+
+  font-size: $text--xl;
+
+  border-radius: $rounded--full;
+
+  span {
+    margin-left: 0.3em; // manually center those stubborn emojis
+  }
+
+  &:hover {
+    background-color: $hover-background-light;
+  }
+
```

Comment:
sure, what should it look like? Do note that the the button remains focused after clicking it though

Reply:
I'd say a simple border/outline would be fine. You can use `:focus-visible` instead of just `:focus` so that the focus state is not visible after clicking the button: https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible